### PR TITLE
ISSUE #1 and #2 - Add Code To Make Corners

### LINKED ISSUE #1: Sharp Corner

### PR DESCRIPTION
**Pull request by 0x6f736f646f**
Signed-off-by: 0x6f736f646f <blackd0t@protonmail.com>
Resolves #1 #2

### CODE REVIEW

File: Code/lib/motor/motor.cpp [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    turn_delay = turndelay;
+    turn_speed = turnspeed;
+
+    // Set all the motor control pins to outputs
+    pinMode(motor_pins.ENA, OUTPUT);
+    pinMode(motor_pins.ENB, OUTPUT);
+    pinMode(motor_pins.IN1, OUTPUT);
+    pinMode(motor_pins.IN2, OUTPUT);
+    pinMode(motor_pins.IN3, OUTPUT);
+    pinMode(motor_pins.IN4, OUTPUT);
```
Comment (SammyOina) [inline Code/lib/motor/motor.cpp:16]: Can you move this to an initialize() function that can be called in the setup function and the initial class initialization can occur in globally outside the setup function

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — DISMISSED** [review]
lgtm

File: Code/lib/motor/motor.cpp [hunk 2]
```diff
@@ -0,0 +1,91 @@
+#include "Arduino.h"
```
Comment (1998-felix) [inline Code/lib/motor/motor.cpp:1]: Do you need to add the arduino header file here?

**Review by 1998-felix — APPROVED** [review]

**Comment by WashingtonKK** [comment]
image

**Comment by WashingtonKK** [comment]
> image

I didn't know you could digitalWrite a value.

File: Code/lib/motor/motor.cpp (hunk 2 above)
Comment (SammyOina) [reply Code/lib/motor/motor.cpp:1]: move it to header file

Comment (0x6f736f646f) [reply Code/lib/motor/motor.cpp:1]: Yes. Because that is where I am using arduino functions

Comment (SammyOina) [reply Code/lib/motor/motor.cpp:1]: Just for uniformity because you're including the header file, it is ok to include Arduino in the header file.

**Review by SammyOina — APPROVED** [review]

**Review by WashingtonKK — APPROVED** [review]

**Pull request merged**